crates/exex/exex/src/event.rs

```diff
@@ -11,3 +11,9 @@ pub enum ExExEvent {
     /// On reorgs, it's possible for the height to go down.
     FinishedHeight(BlockNumber),
 }
+
+impl Default for ExExEvent {
```

Comment:
i'm fine with it, but please implement only for `cfg(test)`

Reply:
Done